src/bot.js

```diff
@@ -85,6 +92,19 @@ export async function initBot() {
 		}
 	}
 
+	// Connect to the database
+	mongoose
+		.connect(MONGODB, {
+			useNewUrlParser: true,
+			useUnifiedTopology: true,
+		})
+		.then((_) => {
+			console.log('Connected to MongoDB');
+		})
+		.catch((err) => {
+			console.log('Unable to connect to MongoDB Database.\nError: ' + err);
```

Comment:
Would it be better to use `console.error` here instead of `console.log`?

Reply:
> Would it be better to use `console.error` here instead of `console.log`?

This is in reference to line 105, not 102

Reply:
Or consider using `winston`!

Reply:
This is a discussion on it self

Reply:
@davidzwa Do you want to add an issue about this and what we talked about over at the Nick bot?